mlflow/tracking/client.py

```diff
@@ -1483,6 +1483,7 @@ def print_artifact_info(artifact):
         """
         return self._tracking_client.list_artifacts(run_id, path)
 
+    @deprecated("mlflow.artifacts.download_artifacts", "2.0")
```

Comment:
The warning message looks incorrect. It should be `mlflow.tracking.client.MflowClient.download_artifacts`

<img width="1095" alt="image" src="https://user-images.githubusercontent.com/17039389/186075991-371b6dae-ea19-433e-a706-7671a6ebe369.png">

Reply:
We can address this issue in a separate PR.

Reply:
I will file a follow-up PR to fix this.

Reply:
You don't need to because I already did in #6552.